### PR TITLE
Update service-tier-hyperscale.md

### PR DESCRIPTION
**Pull request by KoprowskiT**
A few changes with grammar, punctuation and wording.

### CODE REVIEW

File: azure-sql/database/service-tier-hyperscale.md [hunk 1]
```diff
@@ -20,22 +20,22 @@ Azure SQL Database is based on SQL Server Database Engine architecture that is a
...
+The Hyperscale service tier is suitable for all workload types. Its cloud-native architecture provides independently scalable compute and storage to support the widest variety of traditional and modern applications. Compute and storage resources in Hyperscale substantially exceed the resources available in the General Purpose and Business Critical tiers. 
 
 > [!NOTE]
 >
 > - For details on the General Purpose and Business Critical service tiers in the vCore-based purchasing model, see [General Purpose](service-tier-general-purpose.md) and [Business Critical](service-tier-business-critical.md) service tiers. For a comparison of the vCore-based purchasing model with the DTU-based purchasing model, see [Azure SQL Database purchasing models and resources](purchasing-models.md).
-> - The Hyperscale service tier is currently only available for Azure SQL Database, and not Azure SQL Managed Instance.
+> - The Hyperscale service tier is currently only available for Azure SQL Database and not Azure SQL Managed Instance.
```
Comment (dimitri-furman) [inline azure-sql/database/service-tier-hyperscale.md:28]:
```suggestion
> - The Hyperscale service tier is currently only available for Azure SQL Database and not for Azure SQL Managed Instance.
```

File: azure-sql/database/service-tier-hyperscale.md [hunk 2]
```diff
@@ -20,22 +20,22 @@ Azure SQL Database is based on SQL Server Database Engine architecture that is a
...
 
 ## What are the Hyperscale capabilities
 
 The Hyperscale service tier in Azure SQL Database provides the following additional capabilities:
 
-- Rapid Scale up - you can, in constant time, scale up your compute resources to accommodate heavy workloads when needed, and then scale the compute resources back down when not needed.
+- Rapid Scale up - you can constantly scale up your compute resources to accommodate heavy workloads when needed and then scale the compute resources back down when not needed.
```
Comment (dimitri-furman) [inline azure-sql/database/service-tier-hyperscale.md:34]: The original use of "in constant time" is correct. It refers to the fact that each scaling operation takes approximately the same time regardless of database size. That is different from "constantly scale", which refers to scaling operations that occur frequently.

File: azure-sql/database/service-tier-hyperscale.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ For more information about the compute sizes for the Hyperscale service tier, se
 
 ## Who should consider the Hyperscale service tier
 
-The Hyperscale service tier is intended for all customers who require higher performance and availability, fast backup and restore, and/or fast storage and compute scalability. This includes customers who are moving to the cloud to modernize their applications as well as customers who are already using other service tiers in Azure SQL Database. The Hyperscale service tier supports a broad range of database workloads, from pure OLTP to pure analytics. It's optimized for OLTP and hybrid transaction and analytical processing (HTAP) workloads.
+The Hyperscale service tier is intended for all customers who require higher performance and availability, fast backup and restore, and/or fast storage and compute scalability. This includes customers who are moving to the cloud to modernize their applications as well as customers who are already using other service tiers in the Azure SQL Database. The Hyperscale service tier supports a broad range of database workloads, from pure OLTP to pure analytics. It's optimized for OLTP and hybrid transaction and analytical processing (HTAP) workloads.
```
Comment (dimitri-furman) [inline azure-sql/database/service-tier-hyperscale.md:52]: "Azure SQL Database" is the name of a service and as such does not require an article in this context (unlike, for example, a generic "database").

**Review by dimitri-furman — CHANGES_REQUESTED** [review]
Thanks for catching these issues and inconsistencies. Similar to the other PR, I left a couple of suggestions.

**Comment by rwestMSFT** [comment]
Replaced by internal PR https://github.com/MicrosoftDocs/sql-docs-pr/pull/28663.

@KoprowskiT We use the Oxford comma, so I removed a few of your edits. As my colleague also noted, Azure SQL Database is the name of the service, so other edits were redacted.

It will take about a day for the internal PR to be approved and published, at which point you will see the updated content.

Thank you again for your contributions. We appreciate your close attention to our content.

**Pull request closed without merging**